### PR TITLE
Added link to Api documentation (Corrected my mistake)

### PR DESCRIPTION
**Pull request by EmmaSamoilova**

### CODE REVIEW

File: src/const/faqs.jsx [hunk 1]
```diff
@@ -148,9 +148,11 @@ const faqs = [
       </Translation>
     ),
     answer: (
-      <Translation>
-        {(t) => `${t('faq.answer.linkAPI')}`}
-      </Translation>
+      <a href={`${process.env.REACT_APP_API_BASE_URL}schema/redoc/`}>
```
Comment (roman-tiukh) [inline src/const/faqs.jsx:151]: what happen if 
1. REACT_APP_API_BASE_URL = https://ipa.dataocean.us/
2. REACT_APP_API_BASE_URL = https://ipa.dataocean.us

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: src/const/faqs.jsx (hunk 1 above)
Comment (EmmaSamoilova) [reply src/const/faqs.jsx]: 1)  <a href={`${process.env.REACT_APP_API_BASE_URL}schema/redoc/`}>
2) <a href={`${process.env.REACT_APP_API_BASE_URL}/schema/redoc/`}>

File: src/const/faqs.jsx [hunk 2]
```diff
@@ -148,9 +148,11 @@ const faqs = [
       </Translation>
     ),
     answer: (
-      <Translation>
-        {(t) => `${t('faq.answer.linkAPI')}`}
-      </Translation>
+      <a href={`${process.env.REACT_APP_API_BASE_URL}/schema/redoc/`}>
```
Comment (roman-tiukh) [inline src/const/faqs.jsx:151]: 1. https://ipa.dataocean.us/schema/redoc/
2. https://ipa.dataocean.us//schema/redoc/

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: src/const/faqs.jsx (hunk 2 above)
Comment (EmmaSamoilova) [reply src/const/faqs.jsx:151]: In the .env file REACT_APP_API_BASE_URL = https://ipa.dataocean.us. 
 <a href={`${process.env.REACT_APP_API_BASE_URL}/schema/redoc/`}>

**Pull request closed without merging**